### PR TITLE
Added the payment step & query params mapping to state

### PR DESCRIPTION
**Pull request by ErkoRisthein**

### CODE REVIEW

**Comment by ErkoRisthein** [comment]
Still work in progress, ignore hardcoded translations, commented out imports etc.

File: src/newUserFlow/payment/Payment.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React, { PropTypes as Types } from 'react';
+import { connect } from 'react-redux';
+// import { Message } from 'retranslate';
```
Comment (JordanValdma) [inline src/newUserFlow/payment/Payment.js]: commented out code

File: src/newUserFlow/payment/Payment.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+// import { Link } from 'react-router';
+
+export const Payment = ({ userId }) => (
+  <div>
+    <div className="mb-4">
+      <p className="mb-4 mt-5 lead">Liikmetasu</p>
+      <p>Järgmiseks kanna üle liikmetasu ning oledki Tuleva liige.</p>
```
Comment (JordanValdma) [inline src/newUserFlow/payment/Payment.js]: Message?

File: src/exchange/reducer.js [hunk 3]
```diff
@@ -157,6 +159,11 @@ export default function exchangeReducer(state = initialState, action) {
         ...state,
         mandateSigningError: null,
       };
+    case QUERY_PARAMETERS:
```
Comment (JordanValdma) [inline src/exchange/reducer.js]: I would personally create "IS_NEW_MEMBER" dispatch. query params is not domain related abstraction, but implementation detail

Comment (JordanValdma) [inline src/exchange/reducer.js]: Not compulsory ofc

File: src/newUserFlow/payment/Payment.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      <p>Seejärel saad kogu oma pensioni Tulevasse üle tuua.</p>
+
+      <a
+        href={`https://payment-test.maksekeskus.ee/pay/1/link.html?shopId=322a5e5e-37ee-45b1-8961-ebd00e84e209&amount=1000&reference=${userId}`}
```
Comment (JordanValdma) [inline src/newUserFlow/payment/Payment.js]: Going to backend eventually?

**Pull request merged**